packages/frontend/src/index.tsx

```diff
@@ -0,0 +1,21 @@
+import 'normalize.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Pode ser um import assíncrono para acelerar a FCP
```suggestion
import('bootstrap/dist/css/bootstrap.min.css');
```